addons/my-ocular/discuss.css

```diff
@@ -0,0 +1,28 @@
+.postfootright {
+  margin-bottom: 4px; /* so that the buttons have some space */
```

Comment:
Use padding instead of margin so that there is no white space below left part of the post container.

Reply:
thanks